src/Testing.java

```diff
@@ -1,25 +0,0 @@
-import static org.junit.Assert.*;
-
-import org.junit.Test;
-
-public class Testing {
-
-	public class AllTests {
-		@Test
-		public void setUp()throws Exception
-		{
-			
-		}
-		@Test
-		public void testWenjingManalAndAddSeeds()
```

Comment:
Please explain how this test does not add value.  It seems like a valid test.  Is it duplicated somewhere else?  If so, please comment where it is duplicated.